src/css/object.section.css

```diff
@@ -0,0 +1,30 @@
+/* ---------- OBJECT section ---------- */
+
+.section {
+  --section-gutters-y: var(--size-10);
+  padding-bottom: var(--section-gutters-y);
+  padding-top: var(--section-gutters-y);
+  scroll-margin-top: var(--size-scroll-top);
+
+  @media (--mq-sm) {
+    --section-gutters-y: var(--size-12);
+  }
+
+  @media (--mq-md) {
+    --section-gutters-y: var(--size-14);
+  }
+
+  @media (--mq-lg) {
+    --section-gutters-y: var(--size-16);
+  }
+}
+
+.section--full {
```

Comment:
Thought: Curious about the `section`>`wrapper` vs. `wrapper`>`section` approaches on the table.

I guess with this approach of having a `wrapper` around the whole page (i.e. first child of `main`) we'll get forgo the need to nest a `wrapper` in every `section`. Then in which case it's just the `section--full`s that will likely need the inner `wrapper` to suck them back in after they've been pulled out to the viewport width?

Reply:
Exactly, it was simply a numbers game 🙂 

There are way more `section`s that won't be full width so a global `wrapper` made more sense. For the few `section--full`s we'll give them the inner `wrapper`.